spec/services/civil_servant_select2_options_spec.rb

```diff
@@ -18,15 +21,27 @@
       [
         {
           children: [
-            { id: 'example1@example.test', text: 'Zivi Mustermann, example1@example.test' },
-            { id: 'example2@example.test', text: 'Zivi Mustermann, example2@example.test' }
+            {
+              id: first_civil_servant.user.email,
+              text: "#{first_civil_servant.full_name}, #{first_civil_servant.user.email}"
+            },
+            {
+              id: second_civil_servant.user.email,
+              text: "#{second_civil_servant.full_name}, #{second_civil_servant.user.email}"
+            }
```

Comment:
Maybe use something like
```suggestion
              text: "#{all_civil_servants.first.full_name}, #{all_civil_servants.first.user.email}"
            },
            {
              id: second_civil_servant.user.email,
              text: "#{all_civil_servants.second.full_name}, #{all_civil_servants.second.user.email}"
            }
```
to omit the two lets?